GPU/D3D11/DepalettizeShaderD3D11.cpp

```diff
@@ -103,10 +122,17 @@ ID3D11ShaderResourceView *DepalShaderCacheD3D11::GetClutTexture(GEPaletteFormat
 	data.pSysMem = rawClut;
 	// Regardless of format, the CLUT should always be 1024 bytes.
 	data.SysMemPitch = 1024;
```

Comment:
Won't this be incorrect if `expandTo32bit`?  I'm not sure if it matters since there's only 1 row anyway.

-[Unknown]